### PR TITLE
Added info / procedures with screen captures for the Services tab in CF4.5 SUI

### PR DESCRIPTION
**Pull request by suyogsainkar**
Made the following updates in the Services section -
* The name of the tab.
* The appearance and details of the tab with updated description and screen captures.
* The method and icon for accessing the Cockpit console.
* Added short procedures for the following tasks -
  - How to access the HTML 5 console
  - How to set a retirement date for a service
  - How to retire a service
  - How to edit tags
  - How to edit a service
  - How to remove a service
  - How to set ownership of a service
I noticed a few additional changes, including provision to create and delete snapshots which I will address in a separate PR/bug.

Dayle, would you mind reviewing the updates in this PR? Please let me know if you spot anything that needs fixing, otherwise feel free to merge.
MR: https://github.com/ManageIQ/manageiq_docs/pull/328
Preview: file.bne.redhat.com/ssainkar/Red_Hat_CloudForms-4.5-beta-Introduction_to_the_Self_Service_User_Interface-en-US-servicestab.pdf
Thanks!
Suyog

### CODE REVIEW

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 1]
```diff
@@ -27,12 +98,10 @@ Cockpit is an interactive Linux server administrative interface. You can perform
 Cockpit must be pre-configured and running on the virtual machine. 
 ====
 
-As you can see in the following screen capture of the Self Service user interface, the cockpit button is available next to the HTML 5 console button for a virtual machine instance in the *My Services* tab. 
+The option to open the Cockpit console for a virtual machine is available in the image:ssui-access-button.png[Access Button] button, which also houses the option to open HTML5 Console, in the *Resources* section inside the *Services* tab.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [WIP] I've made this edit, but please double-check that this looks OK on your next pass of the chapter.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 2]
```diff
@@ -1,24 +1,95 @@
...
+. Edit the name and description of the service as required.
+image:ssui-edit-service.png[]
+. Click *Save*.
 
 === Cockpit Integration
 
 Cockpit is an interactive Linux server administrative interface. You can perform simple administration tasks such as starting containers, storage administration, network configuration, and inspecting logs. While cockpit allows you to monitor and administer several servers at the same time, your browser connects to a primary server that runs the Cockpit web service through which connections to additional servers can be established.
 
-{product-title} full administrative user interface and the Self Service user interface have enabled for users to click the new Cockpit button image:SSUI_Cockpit-Button.png[Cockpit Button] that opens a new browser window with Cockpit interface for a given virtual machine. 
+{product-title} full administrative user interface and the Self Service user interface have enabled for users to access the Cockpit interface for a given virtual machine.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [DONE] - I made this change.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 3]
```diff
@@ -1,24 +1,95 @@
...
+
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Edit*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Edit*. An *Edit Service* screen appears.
+. Edit the name and description of the service as required.
+image:ssui-edit-service.png[]
+. Click *Save*.
 
 === Cockpit Integration
 
 Cockpit is an interactive Linux server administrative interface. You can perform simple administration tasks such as starting containers, storage administration, network configuration, and inspecting logs. While cockpit allows you to monitor and administer several servers at the same time, your browser connects to a primary server that runs the Cockpit web service through which connections to additional servers can be established.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [DONE] - I made this change.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 4]
```diff
@@ -1,24 +1,95 @@
...
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Remove*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Remove*. A confirmation window appears.
+. Click *Yes, Remove Service*.
+
+=== Editing a Service
+
+Follow this procedure to edit the name and description of a service.
+
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Edit*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Edit*. An *Edit Service* screen appears.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [EDITED] I combined these steps - please feel free to review and restructure if needed.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 5]
```diff
@@ -1,24 +1,95 @@
...
+
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Remove*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Remove*. A confirmation window appears.
+. Click *Yes, Remove Service*.
+
+=== Editing a Service
+
+Follow this procedure to edit the name and description of a service.
+
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Edit*.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [WIP] I've made this edit, but please double-check that the correct meaning is there now.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 6]
```diff
@@ -1,24 +1,95 @@
...
+. To add a tag, select the tag category and value from the lists, then click the blue plus (+) sign as shown in the screen capture.
+. To edit tags, simply change the value assigned to a category, or click on the cross sign in the tag to delete an existing tag.
+. Click *Save*.
+
+=== Removing a Service
+
+Follow this procedure to remove a service. 
+
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Remove*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Remove*. A confirmation window appears.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [EDITED] I combined these steps - please feel free to review and restructure if needed.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 7]
```diff
@@ -1,24 +1,95 @@
...
+. Select an owner from the list.
+. Select a group from the list.
+. Click *Save*.
+
+=== Adding / Editing Tags
+
+Follow this procedure to add or edit tags for a service. 
+
+. From the *Services* screen, click on more options next to the service you want add or edit tags for, then click *Edit Tags*.
+. Alternatively, click on the service. Click the image:ssui-edit-tags-button.png[] button located on the upper-right corner of the screen, then select *Edit Tags*.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [EDITED] I combined these steps - please feel free to review and restructure if needed.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 8]
```diff
@@ -1,24 +1,95 @@
...
+Follow this procedure to set ownership of a service. 
+
+. From the *Services* screen, click on more options next to the service you want set ownership for, then click *Set Ownership*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Set Ownership*.
+image:ssui-set-service-ownership.png[]
+. Select an owner from the list.
+. Select a group from the list.
+. Click *Save*.
+
+=== Adding / Editing Tags
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [DONE] I've edited this.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 9]
```diff
@@ -1,24 +1,95 @@
...
+. Click the image:ssui-retirement-button.png[Service Retire Button] button located on the upper-right corner of the screen, then select *Retire*. The *Retire Service Now* window will appear to confirm your action.
+image:ssui-retire-service-now.png[]
+. Click *Yes, Retire Service Now*.
+
+=== Setting Ownership of a Service
+
+Follow this procedure to set ownership of a service. 
+
+. From the *Services* screen, click on more options next to the service you want set ownership for, then click *Set Ownership*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Set Ownership*.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [EDITED] I combined these steps - please feel free to review and restructure if needed.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 10]
```diff
@@ -1,24 +1,95 @@
...
+. From the *Services* screen, click on the service you want to retire now.
+. Click the image:ssui-retirement-button.png[Service Retire Button] button located on the upper-right corner of the screen, then select *Retire*. The *Retire Service Now* window will appear to confirm your action.
+image:ssui-retire-service-now.png[]
+. Click *Yes, Retire Service Now*.
+
+=== Setting Ownership of a Service
+
+Follow this procedure to set ownership of a service. 
+
+. From the *Services* screen, click on more options next to the service you want set ownership for, then click *Set Ownership*.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [WIP] I added the word "button" so it's "the more options button", but this should still be reviewed, and a UI icon would be good here and in the other instances.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 11]
```diff
@@ -1,24 +1,95 @@
...
+There is now the option to perform most of these tasks in bulk (except editing service details and power operations). In order to do so, select the services and then click the designated button for the required task on the upper-left corner of the screen. 
 
-Moreover, you can now add custom buttons and custom button groups that can have dialogs associated with them. For example:
+Additionally, click on a service to view more details, including information on associated virtual machines and snapshots, and its relationships data.
 
-image:SSUI_Custom_Button1.png[]
+[[html5-console]]
+=== Accessing the HTML5 Console
+
+Follow this procedure to get HTML5 console access to the virtual machine. The console is the remote control system that enables you to work and interact directly with your virtual machines.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [DONE] I removed "work and", as "interact with" seemed more meaningful. If the sentence is now missing crucial meaning, please feel free to reassess.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 12]
```diff
@@ -1,24 +1,95 @@
...
+image:ssui-services-menu.png[]
 
-image:SSUI_Service_Details1.png[]
+There is now the option to perform most of these tasks in bulk (except editing service details and power operations). In order to do so, select the services and then click the designated button for the required task on the upper-left corner of the screen. 
 
-Moreover, you can now add custom buttons and custom button groups that can have dialogs associated with them. For example:
+Additionally, click on a service to view more details, including information on associated virtual machines and snapshots, and its relationships data.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [DONE] - Made this edit.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 13]
```diff
@@ -1,24 +1,95 @@
...
 
-Click a service to view further details including information on the virtual machines associated with it. There is now ability for users to view chargeback data per service for the virtual machines' month-to-date usage. You can also get HTML 5 console access to the virtual machine by clicking the image:SSUI_html5_icon.png[Html 5 Icon] button, as highlighted on the screen capture. On the same screen, you also have options to retire the service now or at a later date, edit or remove the service. Additionally, you can now change the ownership and group of the service using the *Set Ownership* button, and also *Reconfigure* a service, similar to in the full administrative user interface. 
+image:ssui-services-menu.png[]
 
-image:SSUI_Service_Details1.png[]
+There is now the option to perform most of these tasks in bulk (except editing service details and power operations). In order to do so, select the services and then click the designated button for the required task on the upper-left corner of the screen.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [WIP] Edited "There is now the option to perform "
Note - I have changed "left" to "right" as I'm pretty sure this was the intended wording. But if you can double check that would be great. :)

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 14]
```diff
@@ -1,24 +1,95 @@
...
+The *Services* tab lists all available services with a summary of information associated with each service, including the *Service Name*, its current *Power State* represented by an icon, *Number of VMs*, *Owner*, date *Created*, and *Retires* date. Additionally, you can see the chargeback cost per service for a virtual machine's month-to-date usage. You can filter or sort services by those entities, and by *Relative Cost*. 
 
-image:SSUI_MyServices_tab.png[]
+On the same screen, you have the option to edit the service name and description, edit tags, set ownership, retire the service now or at a later date, or remove the service. You can also perform power operations on virtual machines by clicking more options _(represented by three dots)_ next to the service.
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [WIP] I added the word "button" so it's "the more options button", but this should still be reviewed.

File: doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc [hunk 15]
```diff
@@ -1,24 +1,95 @@
 [[my-services-tab]]
 
-== The My Services Tab
+== Services Tab
```
Comment (dayleparker) [inline doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc]: [Done - I've made this edit]

**Comment by dayleparker** [comment]
@suyogsainkar
Hi Suyog,
I've made the smaller/more crucial edits now and will merge this PR, but there are still some things that should be reviewed and fixed. Please let me know if you want to discuss or have questions; happy to review and merge the second iteration of this content as well :)
Thanks so much,
Dayle

**Comment by suyogsainkar** [comment]
@dayleparker Hi Dayle, many thanks for reviewing and merging the content. I will create a separate PR to fix the branding on images, and for the other changes planned for the post-beta version of the guide. :)

**Pull request merged**